sonic-slave/Dockerfile

```diff
@@ -120,6 +120,8 @@ RUN apt-get update && apt-get install -y \
         libcurl3-nss-dev \
         libunwind8-dev \
         telnet \
+        libc-ares2 \
```

Comment:
Why we aren't making this the dependencies of our .deb package? (Install them with our .dev)